
Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-credential-initializer-55n79[0m
{"level":"info","ts":1592912241.653317,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1592912241.6536775,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-working-dir-initializer-w8t8t[0m
{"level":"info","ts":1592912242.9761422,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1592912242.9789014,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-place-tools[0m

Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-ablazleon-jx-py-master-29w9m-dglds[0m
{"level":"info","ts":1592912262.1057372,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1592912262.7975173,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/*********/**-py.git @ v0.0.1 in path /workspace/source"}

Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:
WARNING: no SHAs to merge, falling back to initial cloned commit

Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-jx-git-credentials[0m
Generated Git credentials file /workspace/xdg_config/git/credentials

Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mablazleon-jx-py-master-29w9m-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
error checking push permissions -- make sure you entered the correct tag name, and that you are authenticated correctly, and try again: checking push permission for "https://hub.docker.com//*********/**-py:0.0.1": creating push check transport for https: failed: Get "https://https/v2/": dial tcp: lookup https on 10.100.0.10:53: no such host
[31m
Pipeline failed on stage 'from-build-pack' : container 'step-build-container-build'. The execution of the pipeline has stopped.[0m
